### DISCUSSION TITLE
How to update Chrome Browser when ChromeOS is end-of-life

### DISCUSSION THREAD

**Discussion by Roepertje** [sebanc/brunch#1410] [Q&A] [answered]
I have samus image installed on an older laptop.
Chrome Browser version is now 91.0.4472.167
I can't update ChromeOS any more. **End-Of-Life**!!

Is it possible somehow to update the Chrome Browser?

**Comment by ksoham622** [comment, marked as answer]
Install Rammus if you can.

**Reply by Roepertje** [reply]
I installed now Rammus.
I didn't know Rammus was also for Intel second generation.
I have Sandy Bridge
Chrome Browser is now 97.0.4692.77 

Thanks for your suggestion.

**Reply by uciharis** [reply]
it is work ???
